laa_court_data_api_app/models/hearing_summaries/hearing_summaries_response.py

```diff
@@ -6,4 +6,4 @@
 
 
 class HearingSummariesResponse(BaseModel):
-    hearing_summaries: Optional[list[HearingSummary]] = None
```

Comment:
you missed an Optional import here to remove

Reply:
Done